### PR TITLE
Parameterize XML reports file name

### PR DESCRIPTION
**Pull request by krmahadevan**
Fixes # .

### Did you remember to?

- [ ] Add test case(s)
- [X] Update `CHANGES.txt`

We encourage pull requests that:

* Add new features to TestNG (or)
* Fix bugs in TestNG

If your pull request involves fixing SonarQube issues then we would suggest that you please discuss this with the 
[TestNG-dev](https://groups.google.com/forum/#!forum/testng-dev) before you spend time working on it.

### CODE REVIEW

**Comment by krmahadevan** [comment]
@juherr - This is the PR that is related to my comment [here](https://github.com/cbeust/testng/issues/1218#issuecomment-312573347)

**Review by juherr — APPROVED** [review]
👍 
Ping @cbeust

File: src/main/java/org/testng/reporters/XMLReporter.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 public class XMLReporter implements IReporter {
 
   public static final String FILE_NAME = "testng-results.xml";
+  private static final String JVM_ARG = "xml.report.name";
```
Comment (cbeust) [inline src/main/java/org/testng/reporters/XMLReporter.java]: There's no clear standard for this but I think we should prefix system properties with `testng`, e.g. `testngXmlReportName` or something like that.

Comment (juherr) [inline src/main/java/org/testng/reporters/XMLReporter.java]: Good idea, and maybe `report` before `xml`: `testng.report.xml.name` (or maybe just `testng.report.name`.

Comment (krmahadevan) [inline src/main/java/org/testng/reporters/XMLReporter.java]: @cbeust / @juherr  - I have fixed this by using `testing.report.xml.name`

**Pull request merged**